CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.0)
 
 project(jsoncpp)
 
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
Unrelated to this PR, please revert.

Reply:
If the standard isn't at least 17, MSVC selects 14 as a default and emits a warning saying that [[maybe_unused]] is only available in 17 or greater. I don't think it makes much sense to not set it at all.

Reply:
Then it should be set to 17